bazel/repository_locations.bzl

```diff
@@ -100,6 +100,18 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2019-08-08",
         cpe = "cpe:2.3:a:google:boringssl:*",
     ),
+    aspect_bazel_lib = dict(
+        project_name = "Aspect Bazel helpers",
+        project_desc = "Base Starlark libraries and basic Bazel rules which are useful for constructing rulesets and BUILD files",
+        project_url = "https://github.com/aspect-build/bazel-lib",
+        version = "0.11.10",
```

Comment:
`0.12.0` is out - https://github.com/aspect-build/bazel-lib/releases/tag/v0.12.0

Reply:
the wheel keeps turning!